validator/sawtooth_validator/networking/interconnect.py

```diff
@@ -278,28 +277,30 @@ def _dispatch_message(self):
 
                 zmq_identity, msg_bytes = \
                     yield from self._dispatcher_queue.get()
+
                 message = validator_pb2.Message()
                 message.ParseFromString(msg_bytes)
-                LOGGER.debug("%s receiving %s message: %s bytes",
-                             self._connection,
-                             get_enum_name(message.message_type),
-                             sys.getsizeof(msg_bytes))
 
+                message_type = message.message_type
                 tag = get_enum_name(message.message_type)
+
+                LOGGER.debug("%s receiving %s",
+                             self._connection,
+                             tag)
+
                 self._get_received_message_counter(tag).inc()
 
-                if zmq_identity is not None:
-                    connection_id = \
-                        self._identity_to_connection_id(zmq_identity)
-                else:
-                    connection_id = \
-                        self._identity_to_connection_id(
-                            self._connection.encode())
+                connection_id = \
```

Comment:
I find the previous format easier to read than this inline if across multiple lines.